src/Domain/BoundaryConditions/BoundaryCondition.hpp

```diff
@@ -0,0 +1,34 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <charm++.h>
+#include <memory>
+#include <pup.h>
+#include <string>
+
+#include "Parallel/CharmPupable.hpp"
+
+/// \ingroup ComputationalDomainGroup
+/// \brief %Domain support for applying boundary conditions
+namespace domain::boundary_conditions {
+/*!
+ * \brief Base class off of which all system-specific base classes must inherit.
```

Comment:
delete `of`

Reply:
I don't understand. The current is proper English. I'll change `off of` to `from`.